### DISCUSSION TITLE
Debugging best practices

### DISCUSSION THREAD

**Discussion by taylorpreston** [blitz-js/blitz#2713] [Q&A]
Hey guys! 

I am looking for some tips on the best vscode setup for quick development and debugging. I haven't seen any documentation about it and was wondering if anyone out there has a cheatsheet of sorts I could use. 😄

**Comment by flybayer** [comment]
Hey! I don't have any tips for you (I use vim), but if you figure this out, I'd love if you could PR a new page to our docs for this 💜
